.travis.yml

```diff
@@ -22,11 +23,15 @@ matrix:
       env: TOXENV=py36
     - python: "3.7"
       env: TOXENV=py37
-      dist: xenial
       sudo: true
     - python: "3.7"
       env: TOXENV=py37-nooptionals
```

Comment:
We only need this for the latest.

Reply:
@brian-brazil I'm not sure I understand...
I made this dist (xenial) the default for everything (I think Python 3.8 needs it too).
and just changed the dist for 2.6 to trusty since python 2.6 doesn't work in xenial...

Reply:
We only need the nooptionals for the latest

Reply:
ah! got it.
will move it to 3.8 then!